packages/templates/clients/websocket/java/quarkus/components/QueryParamsVariables.js

```diff
@@ -0,0 +1,32 @@
+import { FormatHelpers } from '@asyncapi/modelina';
+import { Text } from '@asyncapi/generator-react-sdk';
```

Comment:
seems like this can be generalized with https://github.com/asyncapi/generator/blob/master/packages/templates/clients/websocket/python/components/QueryParamsVariables.js

ok if as a followup with instructions for newcomers